lib/foreman_tasks_core/runner/base.rb

```diff
@@ -41,6 +41,16 @@ def close
         # if cleanup is needed
       end
 
+      def timeout
+        # Override when timeouts and regular kills should be handled differently
+        kill
```

Comment:
What about adding a debug message to `@continuous_output` about the timeout passed, something like `Time out for the execution passed, trying to kill` (perhaps the working could be better: it sounds pretty violent when put out of the context :)